### DISCUSSION TITLE
How to Create FastAPI Project Structure via CLI?

### DISCUSSION THREAD

**Discussion by kevinrawal** [fastapi/fastapi-cli#130] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
# I wish I could use something like this:
fastapi create <structure-type>

# Example:
fastapi create standard
```

### Description

Does FastAPI-CLI have a feature to generate a basic structure for creating a FastAPI application? Many other frameworks provide this functionality, making it easier for developers to start with an ideal structure. However, I don't see this feature in FastAPI-CLI, which makes it challenging to begin with a proper structure for FastAPI projects.  

For now, I’ve created my version of this functionality and continue working on it. However, I would like to contribute this feature to FastAPI-CLI. Does this idea make sense?

[Check out my version `type-fastapi`](https://pypi.org/project/type-fastapi/#description) created with [typer](https://typer.tiangolo.com/).

Please let me know if such a feature already exists.
I am willing to contribute this feature to fastapi-cli @tiangolo @Kludex @svlandeg

### Operating System

Other

### Operating System Details

irrelevant 

### Python Version

3.12

**Comment by YuriiMotov** [comment]
There is no such feature.

I believe you already know about [Full Stack FastAPI Template](https://github.com/fastapi/full-stack-fastapi-template)

**Comment by YuriiMotov** [comment]
Let's track it here: https://github.com/fastapi/fastapi-cli/discussions/142
